libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Orchestrator;
+
+namespace Microsoft.Bot.Builder.AI.Orchestrator
+{
+    public class OrchestratorRecognizer : IRecognizer
```

Comment:
>IRecognizer [](start = 42, length = 11)

We are trying to move away from IRecognizer, that's why we added Recognizer.cs and the only reason IRecognizer exists is because we already shipped it.

Reply:
Fixed.